src/modules/elements/time-picker/element.js

```diff
@@ -89,9 +87,9 @@ const TimePicker = ( {
 	);
 
 	const formatLabel = ( seconds ) => {
-		return momentUtil
+		return dateUtil
```

Comment:
```suggestion
		return dateUtil
```
Maybe we abstract out whatever we use inside our "util". This avoids being tied to a particular lib, centralizing a dependency?